tests/TestCase/Database/QueryTest.php

```diff
@@ -2931,6 +2931,29 @@ public function testInsertSimple()
     }
 
     /**
+     * Test insert queries quote integer column names
+     *
+     * @return void
+     */
+    public function testInsertQuoteColumns()
+    {
+        $this->loadFixtures('Articles');
+        $query = new Query($this->connection);
+        $query->insert([123])
+            ->into('articles')
+            ->values([
+                123 => 'mark',
```

Comment:
Even if a column name is numeric shouldn't one use string `'123'` for it? It's silly to use integer to represent a column name IMO.